File: 01.ProgrammingFundamentals/Java/02.ConditionalStatementsAndLoops/Exercises/src/P05_WordInPlural.java
import java.util.Scanner;
public class P05_WordInPlural {
    public static void main(String[] args) {
        Scanner console = new Scanner(System.in);
        String word = console.nextLine();

        if(word.endsWith("y")) {
            System.out.println(word.substring(0,word.length()-1) + "ies");
        } else if(word.endsWith("o") || word.endsWith("ch") || word.endsWith("s") ||
                word.endsWith("sh") || word.endsWith("x") || word.endsWith("z")) {
            System.out.println(word + "es");
        } else {
            System.out.println(word + "s");
        }
    }
}

File: ProgrammingFundamentals/C#/02.ConditionalStatementsAndLoops/Exercises/P05_WordInPlural/P05_WordInPlural.cs
using System;

namespace P05_WordInPlural
{
    class P05_WordInPlural
    {
        static void Main(string[] args)
        {
            String word = Console.ReadLine();

            if (word.EndsWith("y"))
            {
                word = word.Remove(word.Length - 1);
                Console.WriteLine(word + "ies");
            }
            else if (word.EndsWith("o") || word.EndsWith("ch") || word.EndsWith("s") ||
                  word.EndsWith("sh") || word.EndsWith("x") || word.EndsWith("z"))
            {
                Console.WriteLine(word + "es");
            }
            else
            {
                Console.WriteLine(word + "s");
            }
        }
    }
}
